ruby/ql/lib/codeql/ruby/typetracking/TypeTracker.qll

```diff
@@ -8,22 +8,22 @@ private module Cached {
    * A description of a step on an inter-procedural data flow path.
    */
   cached
-  newtype TStepSummary =
```

Comment:
I see that most/all things in this module are deprecated. Shall we add a deprecation note to the file-level QL doc explaining what to use instead?

Reply:
Good idea.